common/actions/config/actionTypes.ts

```diff
@@ -61,25 +61,25 @@ export interface ChangeNetworkRequestedAction {
 /*** Add Custom Node ***/
 export interface AddCustomNodeAction {
   type: TypeKeys.CONFIG_ADD_CUSTOM_NODE;
-  payload: { id: string; config: CustomNodeConfig };
+  payload: CustomNodeConfig;
 }
 
 /*** Remove Custom Node ***/
 export interface RemoveCustomNodeAction {
   type: TypeKeys.CONFIG_REMOVE_CUSTOM_NODE;
-  payload: { id: string };
+  payload: string;
```

Comment:
Not sure how i feel about changing this to not be object based, it loses context of what you're supposed to pass in the action creator when you use it.

Reply:
Do you think typing this as `CustomNodeConfig['id']` alleviate that? All of the object creation / destructuring was getting on my nerves between this and `removeCustomNetwork`

Reply:
its not that, its when you call an action creator such as `removeCustomNode(myVar)` then its not easy at a quick glance to see what's passed in and why. `removeCusomNode({id: myVar})` makes it more legible and keeps context imo

Reply:
I think the current invocations are clear from a reading standpoint, since they're always `*.id`, here are all of their uses

> yield put(removeCustomNetwork(customNetwork.id));
> removeCustomNetwork(firstCustomNetwork.id)

> this.props.removeCustomNode(this.props.node.id);
> removeCustomNode(firstCustomNode.id)

But yeah, from someone looking to call it for the first time, they might not be sure what to pass. Maybe the best alternative is to pass in the entire config, rather than a string id.